test/integration/collections_integration_test.rb

```diff
@@ -13,7 +13,6 @@ class CollectionsIntegrationTest < ActionDispatch::IntegrationTest
     assert_response :success
 
     assert_select 'title', count: 1, text: "#{@collection.name} | #{@site.name}"
-    assert_select 'div.hero h4', text: 'The Community Calendar'
```

Comment:
Should we check tagline or default description here or is the h4 gone now?

Reply:
So this is testing the raw https://placecal.org/partner/partnername page with no subdomain. Partners should never be viewed in this context really, at least with our current logic -- perhaps we could make a kind of global index page later but yeah. This is what I mean from tests needing a bit of a top down reorganise.